example/21_gemm_normalize_xdl/gemm_layernorm_xdl_fp16.cpp

```diff
@@ -0,0 +1,331 @@
+#include <iostream>
```

Comment:
this folder name should be ```21_gemm_layernorm```

Reply:
2bd585e4fb05f581295681dff0684fd8c18f748b